mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,16 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a, b, c) {
   return a + b + c;
 }
 
-function introduceMe(name, age)
-  return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age) {
+  return `Hello, my name is ${name} and I am ${age} years old`;
```

Comment:
Nice use of string iterpolation!

Reply:
Hi @jdbevan , many thanks for your feedbacks.
The 4-tax has been updated. I still don't understand why it was missing. Thanks again!